karrot/history/models.py

```diff
@@ -43,6 +43,7 @@ class HistoryTypus(enum.Enum):
     ACTIVITY_TYPE_CREATE = 25
     ACTIVITY_TYPE_MODIFY = 26
     ACTIVITY_TYPE_DELETE = 27
+    USER_LOST_EDITOR_ROLE = 18
```

Comment:
I just noticed this number is not unique, I guess was supposed to be `28` :)

Reply:
Oh no 🤦 I'll fix that tomorrow in a new PR